ci/ray_ci/tests.env.Dockerfile

```diff
@@ -18,13 +18,16 @@ RUN <<EOF
 set -euo pipefail
 
```

Comment:
probably also don't need to install dashboard, so move the if condition and exit early here?